### ISSUE TITLE
Regression in AppKit Unity 1.1: The type or namespace name 'Sentry' could not be found

### ISSUE THREAD

**Issue by bibotter** [reown-com/reown-dotnet#22]
Got this error which stops Unity compiling after upgrade from "/com.reown.appkit.unity@1.0.1" => /com.reown.appkit.unity@1.1.0/:
>       Library/PackageCache/com.reown.appkit.unity@1.1.0/Runtime/Controllers/SiweController.cs(5,7): error CS0246: 
>       The type or namespace name 'Sentry' could not be found (are you missing a using directive or an assembly reference?)

Seems like this is a regression last resolved 6 months ago in [https://github.com/WalletConnect/WalletConnectUnity/discussions/214](https://github.com/WalletConnect/WalletConnectUnity/discussions/214) by @skibitsky  :
      "We use Sentry during internal QA. It logs all exceptions that occur during testing. 
      The sample provided with the Modal package doesn't send us any data when you run it."

One workaround for now seems to be to install Sentry, which I did:
       https://docs.sentry.io/platforms/unity/
       https://github.com/getsentry/unity.git#2.3.0  

Apparently WebGL builds dont even support the new "Siwe" ("Sign-in With Ethereum"):
https://github.com/reown-com/reown-dotnet/releases/tag/com.reown.appkit.unity%2F1.1.0
https://docs.reown.com/appkit/unity/core/siwe

So for me another workaround which I'm using right to exclude the only dependency for "using Sentry.Protocol" on the only platform I care about (WebGL):
```
#if !UNITY_WEBGL
using Sentry.Protocol;
#endif

```
I really want to exclude every useless dependency because I dont want a singly byte of unused bloat on WebGL (nobody wants that). For that reason I also made all the bloated Resources/-calls to be loaded asynchronously so they dont get included in the initial WebGL runtime.

PS: Your documentation says "IL2CPP code stripping level: Minimal (or lower)" is a requirement:
[https://docs.reown.com/appkit/unity/core/installation](https://docs.reown.com/appkit/unity/core/installation)

I've taken a deep dive in the code and done a bunch of test on WebGL on the latest Unity 2022.3.x version, and so far it seems that fullfilling this requirement has no effect on Unity WebGL builds other than adding significant bloat to the builds! Stripping high worked just fined and reduced my runtime size by a huge margin.

 If stripping=minimal is required only on the other platforms, it would be worthwhile to document this properly. If my observations are not accurate for some cases, please document what those cases are so I could ensure the dependencies are added manually?

**Comment by skibitsky** [comment]
Hello, 

The Sentry issue has been fixed in v1.1.1

**Comment by skibitsky** [comment]
> If stripping=minimal is required only on the other platforms, it would be worthwhile to document this properly. 

This is a good point. I did some testing with different code stripping levels in WebGL, and indeed, the high stripping level is more stable than on the others on this platform. However, I experienced some issues with serialization when handling certain EVM interactions or building typed signature requests using Nethereum in our sample app [here](https://github.com/reown-com/reown-dotnet/blob/782d831e71e282b7c922b2c777aa66337d058ed7/sample/Reown.AppKit.Unity/Assets/Scripts/Dapp.cs#L365).

That said, some of these issues should be relatively straightforward to address on our end. What types of requests are you interested in? They may already be compatible with high-level code stripping, but if not, maybe I could prioritize improving compatibility for the next release.
